### PR TITLE
Removed the usage of diffForHumans

### PR DESCRIPTION
**Pull request by stromblom**
As discussed, the diff format is not really helpful. And the hover does not work on phones.

### CODE REVIEW

File: app/resources/views/pages/dominion/advisors/rankings.blade.php [hunk 1]
```diff
@@ -87,7 +87,7 @@
                 </div>
                 <div class="box-body">
                     <p>The rankings advisor tells you how well all the dominions are doing in the world.</p>
-                    <p>Rankings are updated every day. Current displayed rankings are from {{ today()->diffForHumans() }}.</p>
+                    <p>Rankings are updated every day. Current displayed rankings are from {{ today() }}.</p>
```
Comment (stromblom) [inline app/resources/views/pages/dominion/advisors/rankings.blade.php]: This still shows the wrong hour.

**Comment by WaveHack** [comment]
Hover not working is because we're still using `<abbr title="foo">bar</abbr>`, instead of the newly preferred `<span data-toggle="tooltip" data-placement="top" title="foo">bar</span>`

File: app/resources/views/pages/dominion/advisors/rankings.blade.php (hunk 1 above)
Comment (WaveHack) [reply app/resources/views/pages/dominion/advisors/rankings.blade.php]: This is already fixed in develop, so no worries there 😄

**Comment by WaveHack** [comment]
I know we discussed it, but I'm still not really 100% on this change. I'll merge it nonetheless.

Absolute timestamps are noisy and quite verbose to read imo. Also due to the lack of (user) locale settings, every timestamp will be in `Y-m-d H:i:s` format until then.

**Comment by WaveHack** [comment]
I have an idea. What about adding a user setting for this? "Show relative times", default enabled (because I believe it improves UX, especially for newer people), which then uses `->diffForHumans()` everywhere (with a proper mobile-working tooltip hover for abs timestamp)? And if disabled it shows abs timestamp everywhere (like in this PR)?

**Comment by internetfett** [comment]
> I have an idea. What about adding a user setting for this? "Show relative times", default enabled (because I believe it improves UX, especially for newer people), which then uses `->diffForHumans()` everywhere (with a proper mobile-working tooltip hover for abs timestamp)? And if disabled it shows abs timestamp everywhere (like in this PR)?
> 
> So people can decide for themselves
> 
> I really like relative timestrings tbh 😅

I think fixing the tooltips would be a great solution even without a user setting.

**Comment by WaveHack** [comment]
This was discussed quite a bit on the Discord previously. Some people really are in favor for absolute times (or even round times).

I think having a setting with the following options to choose from would help everyone:

- Relative time (absolute on hover)
- Absolute time (no hover)
- Round time (like 'Day 5 Hour 23', absolute on hover)

Thanks for the PR, but I'm going to close it for now. I'll be fixing mobile hovers and keeping relative time atm. I'll make another issue to add a time setting for this.

**Comment by stromblom** [comment]
I don't agree  at all. The nondiff is a better standard. I do agree it should be a setting. But I think that could be changed at a later date.

**Comment by WaveHack** [comment]
You know what, you make a good point. I'll merge it

**Pull request merged**